### ISSUE TITLE
I can't unfreeze my TRX, 

### ISSUE THREAD

**Issue by mugman325** [tronscan/tronscan-frontend#655]
at first I tried to freeze my TRX that earned by vote, 
but My tron power is not increased by something went error, 
TRX is still in my wallet, tron power is same as before I tried to freeze, 

now I can't unfreeze my trx, 
or freeze trx, (ERROR message pops up)
unfreeze expire day is still 2018.9.26,  (today is 2018,9.28)

**Comment by paulmeade7** [comment]
Same with my account and I don't even know where to go to get help regarding getting this fixed its been a full day now and nothing works.

**Comment by blcavalcanti** [comment]
I'm having the same problem all day, I can not unfreeze my trx

**Comment by JWhiteUX** [comment]
My wallet says I have TRX, but I can't find where to unlock/freeze the tokens used to vote.

**Comment by JWhiteUX** [comment]
<img width="1219" alt="screen shot 2018-11-12 at 3 33 37 pm" src="https://user-images.githubusercontent.com/12961108/48381341-5c1ec680-e690-11e8-9afe-7cb43ce90c9e.png">
This doesn't make sense because I have TRX, my balance has a value, but I have ZERO (0) TRON power (because I voted), but the system doesn't see the votes?

**Comment by danny1199** [comment]
I also cannot unfreeze my Tron Power.  I can see what is there but it won't unfreeze, despite being long past the three day limit. Has anybody received a solution for this?

**Comment by JWhiteUX** [comment]
This issue got closed with no resolution? DAFUK?

**Comment by Rovak** [comment]
This issue is 5 months old, please use tronscan.org. desktop client is no longer maintained

**Comment by JWhiteUX** [comment]
ok, I'll give it a try. Thanks for the update. Didn't realise the desktop client was no longer maintained

**Comment by wongzhunhou** [comment]
i am using tronscan.org but it is still having these problems. I cant unfreeze it even though it is way pass the 3 days mark. Please help to solve it

**Comment by SkyDex7** [comment]
I just tried using tronscan.org and it won't unfreeze my TRX. The freeze works, I just did it, but when I click unfreeze button the whole page simply turn into a semi-transparent red layer and stays like that without opening unfreeze option. Also, I've passed 3 days mark a month ago, so it's not that.

I can see my account, I can even freeze, but cannot unfreeze. 

Could you please look into it and help me out?

**Comment by wongzhunhou** [comment]
I tried using a different browser and it works, I was using Chrome and I
have that problem. Try it and see if it works for you

**Comment by SkyDex7** [comment]
Hey, wongzhunhou,

Many thanks for the tip, appreciate it! I've tried opening tronscan.org on
windows explorer, but it didn't work either.
But I finally found the workaround.

I logged into my TronWalletPlay account, added my tronscan account and
connected to it via ledger.
I was able to import my tronscan wallet, and ultimately the unfreeze option
worked out smoothly here. I got back to tronscan.org to check it out and my
frozen TRX are now unfrozen.
All's fine now.

Thanks mate!
